runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -576,9 +576,32 @@ bool J9::ValuePropagation::transformUnsafeCopyMemoryCall(TR::Node *arraycopyNode
       TR::TreeTop *tt = _curTree;
       TR::Node *ttNode = tt->getNode();
 
+#if defined(J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION)
+      if (TR::Compiler->om.isOffHeapAllocationEnabled()
+            && (ttNode->getOpCodeValue() == TR::treetop || ttNode->getOpCode().isResolveOrNullCheck()))
+         {
+         OMR::ValuePropagation *vp = this;
+         ListIterator<OMR::ValuePropagation::TR_TreeTopNodePair> copyMemoryIt(&vp->_offHeapCopyMemory);
```

Comment:
Were you getting duplicate trees added to the list? We should reach here at most once per node because `canRunTransformToArrayCopy()` requires `lastTimeThrough()`

Reply:
No but saw that other delayedTransformation do duplicate check. If not needed I'll remove it 👍